android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java

```diff
@@ -116,7 +116,6 @@ public void handleVisibility() {
     @Override
     public void onDestroyView() {
         super.onDestroyView();
-
```

Comment:
@simarsingh24 It's advisable to not do changes that are not related to the issue. Even though this one is very small and hence isn't a problem but it can be a problem if done on a large scale. In case you want to make changes, opening an issue would be the recommended way. Just something to keep in mind. :smile:

Reply:
Yeah totally! I will keep that in mind :)